lib/keyboard.dart

```diff
@@ -65,7 +65,7 @@ class Keyboard extends StatelessWidget {
     return Container(
       width: keyboardSize.width,
       height: keyboardSize.height,
-      margin: EdgeInsets.only(top: 16),
+      margin: EdgeInsets.only(top: 26),
```

Comment:
hi, Thanks for the contribution. 
Can you please revert this change, since it's not related to a "canPop" functionality?